src/include/Makefile.am

```diff
@@ -23,7 +23,10 @@ smbios_c_config_HEADERS = \
                 src/include/smbios_c/config/platform/win32.h \
                 src/include/smbios_c/config/platform/linux.h \
                 src/include/smbios_c/config/abi_prefix.h \
-                src/include/smbios_c/config/get_config.h \
+                src/include/smbios_c/config/get_config.h
+
+smbios_c_compilerdir = $(includedir)/smbios_c/compiler

```

Comment:
This needs to be `$(includedir)/smbios_c/config/compiler` no?

Reply:
Yeah, that's why I shouldn't look at things like this right before bed ;)